src/components/home-page/about-section/index.tsx

```diff
@@ -1,8 +1,14 @@
 import Card from 'components/common/card';
-
 const AboutSection = () => (
-    <section title="about" className="px-4 sm:px-8 md:px-16 lg:px-32 xl:px-48">
-        <div className="flex h-auto flex-col">
+    <section
+        title="about"
+        className=" flex mt-36 md:mt-0 bg-black px-4 sm:px-8 md:px-16 lg:px-32 xl:px-48"
+    >
+        <div className="order-2 -mt-36 ml-7 md:-ml-52 md:mt-0 opacity-75 bg-blend-overlay w-3/4 md:w-1/2 absolute md:relative">
+            <img src="/back.webp" alt="" />
```

Comment:
I'd say we should try to stick to Next.js own `Image` component instead of the generic `img`. Other than that, LGTM.

Reply:
@HaresMahmood Let's do it as part of a separate PR. Approving this one for now.